src/find/matchers/mod.rs

```diff
@@ -348,6 +393,33 @@ fn build_matcher_tree(
                 i += 1;
                 Some(NewerMatcher::new(args[i])?.into_box())
             }
+            "-newerat" | "-newerBt" | "-newerct" | "-newermt" => {
```

Comment:
This covers "only" a part of the possible combinations. From the man page:
```
-newerXY reference

Succeeds  if  timestamp  X  of the file being considered is newer than timestamp Y of the file reference.
The letters X and Y can be any of the following letters:

a   The access time of the file reference
B   The birth time of the file reference
c   The inode status change time of reference
m   The modification time of the file reference
t   reference is interpreted directly as a time

Some combinations are invalid; for example, it is invalid for X to be t. [...]
```
This means you can also have combinations like `-neweraa`, `-neweraB`, `-newerac`, `-neweram`, etc.

Reply:
Sorry, I misunderstood the documentation. I'll rewrite the code.

Reply:
I have completed the implementation of the parameter function described in the documentation and added a portion of the test code. Should we start the review?